web/channels/retro_channel.ex

```diff
@@ -35,6 +35,11 @@ defmodule RemoteRetro.RetroChannel do
     {:noreply, socket}
   end
 
+  def handle_in("user_typing_action_item", %{"userToken" => userToken}, socket) do
```

Comment:
hmmm... this is identical to the `"user_typing_idea"` handler defined up above. not sure we need both, because an action item *is* an idea.